index.js

```diff
@@ -78,10 +98,19 @@ module.exports.convert = function (inFormat, outFormat, content, cb, argumentsFr
   // as an intermediate
 
   // Create the callback for the second half first, for the first half to call
-  var internalCallback = function (data) {
-    // The result of this calls their callback
-    executeApplication(binaryPath(formats.constants.from.METADATA_OBJECT_DESCRIPTION_SCHEMA, outFormat), data, cb, argumentsTo);
+  var internalCallback = function (err, data) {
+    if (!err) {
+      // The result of this calls their callback
+      executeApplication(binaryPath(formats.constants.from.METADATA_OBJECT_DESCRIPTION_SCHEMA, outFormat), data, cb, argumentsTo);
+    }
```

Comment:
Looks to me like if there is an error in the `inFormat -> MODS conversion`, it is swallowed, and the user's callback is never called, when really we should be calling their callback with the error generated?

I'd expect to see this:

```suggestion
    } else {
        cb(err);
    }
```